bowtie/templates/report.html.j2

```diff
@@ -410,13 +419,93 @@
   <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/js/bootstrap.bundle.min.js" integrity="sha384-OERcA2EqjJCMA+/3y+gxIOqMEjwtxJY7qPCqsdltbNJuaOe923+mo//f6V8Qbsw3" crossorigin="anonymous"></script>
   <script>
     const tooltipTriggerList = document.querySelectorAll('[data-bs-toggle="tooltip"]')
     const tooltipList = [...tooltipTriggerList].map(tooltipTriggerEl => new bootstrap.Tooltip(tooltipTriggerEl))
   </script>
-
+  {# Scripts for Schema Display start  #}
+  <script>
+      const displayBody = document.querySelector("#display-body");
+      const codes = document.querySelector("#code");
+      function displayCode(code) {
+        displayBody.classList.remove("d-none");
+        displayBody.classList.add("d-block");
+        codes.innerHTML = `<pre>${code}</pre>`;
+      }
+      function schemaDisplay(parentBodyId, code) {
+        displayBody.classList.remove("d-none");
+        displayBody.classList.add("d-block");
+        displayCode(code);
+        const isDisplayed = displayBody.parentElement === document.getElementById(parentBodyId);
+        // If the code segment is displayed, remove it
+        if (isDisplayed) {
+          displayBody.parentElement.removeChild(displayBody);
+        } else {
+          // If the code segment is not displayed, append it to the accordion body
+          const accordionBody = document.querySelector(`#${parentBodyId}`);
+          accordionBody.insertBefore(displayBody, accordionBody.firstChild);
+        }
+      }
+      {# Close Schema Card  #}
+      function closeSchemaCard(){
+        displayBody.classList.remove("d-block");
+        displayBody.classList.add("d-none");
+      }
+      {# Copy Schema from display  #}
+      function copySchemaFromDisplay(){
+        const textToCopy = document.getElementById("code").innerText;
+        const tempElement = document.createElement("textarea");
+        tempElement.value = textToCopy;
+        document.body.appendChild(tempElement);
+        tempElement.select();
+        tempElement.setSelectionRange(0, 99999);
+        document.execCommand("copy");
+        document.body.removeChild(tempElement);
+        {# Change the Copy Button icon after copied #}
+        {# Checkmark SVG Icon  #}
+        document.getElementById("copy-button").innerHTML = `<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-check" viewBox="0 0 16 16">
+                                                              <path d="M10.97 4.97a.75.75 0 0 1 1.07 1.05l-3.99 4.99a.75.75 0 0 1-1.08.02L4.324 8.384a.75.75 0 1 1 1.06-1.06l2.094 2.093 3.473-4.425a.267.267 0 0 1 .02-.022z"/>
+                                                            </svg>`;
+setTimeout(() => {
+  {# Clipboard SVG Icon setting as a default again after copy  #}
+  document.getElementById("copy-button").innerHTML = `<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-clipboard" viewBox="0 0 16 16">
```

Comment:
I think rather than continuing to copy paste SVG elements, let's just use [Bootstrap Icons](https://icons.getbootstrap.com/#icon-font) at this point? Which I think already has most of the SVGs both that were already present as well as the new ones you're adding.

Reply:
In the html report, every SVG icon is from bootstrap. Yes we can Directly use CDN for icons and, Instead of using the svg code we will use icons like `<i class="bi bi-0-circle"></i>` this.

Reply:
Ok let's decide in a follow-up and stick as-is for now.